packages/rum/src/domain/record/record.ts

```diff
@@ -15,8 +15,8 @@ import type {
 } from '../../types'
 import { RecordType, IncrementalSource } from '../../types'
 import { serializeDocument, SerializationContextStatus } from './serialize'
-import { initObservers } from './observers'
-import type { InputCallback } from './observers'
+import { initObservers } from './observers/observers'
+import type { InputCallback } from './observers/observers'
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏`observers/observers` feels a bit odd, what about having an `index.ts` for all the things that we want to expose from those observers?